src/Http/Http.Features/src/IFormCollection.cs

```diff
@@ -90,5 +90,13 @@ public interface IFormCollection : IEnumerable<KeyValuePair<string, StringValues
         /// </summary>
         /// <returns>The files included with the request.</returns>
         IFormFileCollection Files { get; }
+
+#if NETCOREAPP
+        bool TryGetContentType(string sectionName, out StringValues contentType)
```

Comment:
content-type should just be string, it's a single value header. We do the same thing on HttpRequest/Response.